pkg/docker/docker.go

```diff
@@ -544,8 +545,8 @@ func runContainerTar(opts RunContainerOptions, imageMetadata *docker.Image) (cmd
 	return cmd, tarDestination
 }
 
-// this funtion simply abstracts out the running of the newly produced image when the --run=true option is provided
-func runContainerDockerRun(container *docker.Container, d *stiDocker, image string) {
+// dumpContainerInfo dumps information about a running container (port/IP/etc)
```

Comment:
https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences

We're missing the period.